spec/integration/api_notice_search_spec.rb

```diff
@@ -146,9 +146,7 @@
             {
               description: "The Lion King on YouTube",
               infringing_urls_attributes: [
-                { url: "http://youtube.com/bad_url_3" },
-                { url: "http://youtube.com/bad_url_2" },
-                { url: "http://youtube.com/bad_url_1" }
+                { domain: 'youtube.com', count: 3 }
```

Comment:
Let's also test for the case where users are authenticated and make sure they get the full URLs.

Reply:
Ok